src/main/java/it/gov/pagopa/pu/classification/service/AssessmentsServiceImpl.java

```diff
@@ -0,0 +1,75 @@
+package it.gov.pagopa.pu.classification.service;
+
+import it.gov.pagopa.pu.classification.connector.debtposition.DebtPositionTypeOrgService;
+import it.gov.pagopa.pu.classification.connector.debtposition.InstallmentNoPIIService;
+import it.gov.pagopa.pu.classification.connector.processexecutions.IngestionFlowFileService;
+import it.gov.pagopa.pu.classification.enums.AssessmentStatus;
+import it.gov.pagopa.pu.classification.model.Assessments;
+import it.gov.pagopa.pu.classification.repository.AssessmentsRepository;
+import it.gov.pagopa.pu.debtposition.dto.generated.DebtPositionTypeOrg;
+import it.gov.pagopa.pu.debtposition.dto.generated.InstallmentNoPIIResponse;
+import it.gov.pagopa.pu.p4paprocessexecutions.dto.generated.IngestionFlowFile;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Objects;
+
+@Lazy
+@Slf4j
+@Service
+public class AssessmentsServiceImpl implements AssessmentsService {
+
+    private final InstallmentNoPIIService installmentNoPIIService;
+    private final IngestionFlowFileService ingestionFlowFileService;
+    private final DebtPositionTypeOrgService debtPositionTypeOrgService;
+    private final AssessmentsRepository assessmentsRepository;
+    public static final String CURRENT_DATE_STRING = LocalDate.now().format(DateTimeFormatter.ofPattern("yyyyMMdd"));
```

Comment:
you have not to store the current date! otherwise you will always use the build date!
you should store as constant just the DateTimeFormatter!